src/App/pages/RepoPage/RepoPage.tsx

```diff
@@ -0,0 +1,22 @@
+import Header from '../../../components/Header';
+import Search from './components/Search';
+import Users from './components/Users'
+import React from 'react'
+const RepoPage = () => {
+
+    return (
+
+        <React.Fragment >
+            <Header></Header>
```

Comment:
можно поправить везде

```suggestion
            <Header />
```